controllers/provisioners/ekscloudformation/ekscloudformation.go

```diff
@@ -488,3 +493,22 @@ func (ctx *EksCfInstanceGroupContext) processParameters() error {
 	ctx.parseTags()
 	return nil
 }
+
+//GetManagedPolicyARNs constructs managed policy arns
+func GetManagedPolicyARNs(pNames []string) string {
+	//This is for Managed Policy ARN list.
+	//First add the DEFAULT required policies to the list passed by user as part of custom resource
+	var iamManagedPolicyARNs []string
+	names := strings.Split(reqPolicies, ",")
+	for _, name := range names {
+		iamManagedPolicyARNs = append(iamManagedPolicyARNs, fmt.Sprintf("arn:aws:iam::aws:policy/%s", name))
+	}
+	// Add the user supplied policy names if any
+	if len(pNames) != 0 {
```

Comment:
may be nice to add a check here for `HasPrefix` for `arn:aws:iam::aws:policy`, that way, we are protected if user has already provided full policy name `arn:aws:iam::aws:policy/some-policy` in CR

You can also add another unit test around this

Reply:
Good point. I will add that